sdk/core/azure-core/azure/__init__.py

```diff
@@ -1 +1 @@
-__path__ = __import__('pkgutil').extend_path(__path__, __name__)  # type: ignore
+__path__ = __import__('pkgutil').extend_path(__path__, __name__) # type: str
```

Comment:
`__path__` is an optional iterator of string, but surely not a string itself
https://docs.python.org/3/reference/import.html#__path__

This is one of the few you can ignore, since it brings no value and is not shipped to customer (this file is only on Github)